### ISSUE TITLE
FIX Folder Permissions 

### ISSUE THREAD

**Issue by Heptanas** [BiglySoftware/BiglyBT-Android#248]
On android 12 TV in permission there is no setting to allow management of all files. In older version it is. Please fix it :)

**Comment by Wormsworth** [comment]
Yes,  please! Files are not being promoted between directories. 
Inprocess complete and removed

**Comment by Heptanas** [comment]
So when will fix it?

**Comment by Wormsworth** [comment]
Waiting 2 months- is BBT still being supported?

**Comment by parg** [comment]
Yes, by a very busy dev. Feel free to contribute

**Comment by Wormsworth** [comment]
Still waiting!

**Comment by parg** [comment]
So you are

**Comment by Wormsworth** [comment]
If I could help I would.  The only programing language I know is COBOL.

**Comment by antonsoroko** [comment]
Same issue on android tv 11.
